product-service/src/functions/createProduct/handler.ts

```diff
@@ -0,0 +1,86 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, PutCommand } from '@aws-sdk/lib-dynamodb';
+import { v4 as uuidv4 } from 'uuid';
+import type { IProduct, IProductStock, IProductWithStock } from 'src/types/product';
+import { APIGatewayProxyResult } from 'aws-lambda';
+
+const client = new DynamoDBClient();
```

Comment:
you will use same client between different handlers, maybe make sense to store it somewhere to avoid recreating each time?